src/main/java/com/lambdaherding/edi/ch02/ThreadLocalExample.java

```diff
@@ -9,6 +9,26 @@
 /**
  * This class is not thread-safe! Turn the static formatter into a ThreadLocal version of it,
  * using lambdas for simplicity.
+ * 
+ * Comparrison Output:
+ * 
+ * Thread safe:
+ * <code>
+ * 02-03-2017
+ * 01-03-2017
+ * 01-04-2017
+ * 03-03-2017
+ * 30-06-2064
+ * 30-09-2111
+ * </code>
+ * 
+ * Not thread safe:
+ * 01-03-2017
+ * 02-03-2017
+ * 01-03-2017
+ * 01-04-2017
+ * 01-04-2017
+ * 01-04-2017
```

Comment:
Of course you'll get different results between runs, but the important part is that we have 4 `01-04-2017` entries 2 `01-03-2017` entries and a single `02-03-2017` entry when they should all be unique.